aws/cloudfront/s3.tf

```
@@ -3,7 +3,7 @@ resource "aws_s3_bucket_policy" "asset_bucket" {
 
   policy = <<POLICY
 {
-   "Version":"2021-03-23",
+   "Version":"2008-10-17",
```

```suggestion
   "Version":"2012-10-17",
```
is the current version